python/paddle/v2/fluid/evaluator.py

```diff
@@ -204,3 +204,37 @@ def eval(self, executor, eval_program=None):
             [precision], dtype='float32'), np.array(
                 [recall], dtype='float32'), np.array(
                     [f1_score], dtype='float32')
+
+
+class EditDistance(Evaluator):
+    """
+    Average edit distance error for multiple mini-batches.
```

Comment:
Need more comments, How to usage, and what the returned value by `eval` means.

Reply:
Thk. Done.